lightning/src/ln/msgs.rs

```diff
@@ -257,6 +273,10 @@ pub struct FundingCreated {
 	pub funding_output_index: u16,
 	/// The signature of the channel initiator (funder) on the initial commitment transaction
 	pub signature: Signature,
+	/// The partial signature of the channel initiator (funder)
+	pub partial_signature_with_nonce: Option<PartialSignatureWithNonce>,
```

Comment:
can be called `initiator_partial_signature_with_nonce` even if more verbose than spec